docker-compose-example.yml

```
@@ -78,12 +78,11 @@ services:
       - graphql-engine
     ports:
       - 8545:8545
-      - 3331:3331
     extra_hosts:
       - host.docker.internal:host-gateway
     environment:
       - SUBQL_URL=http://graphql-engine:3001
       - ENDPOINT_URL=ws://mandala-node:9944
       - LOCAL_MODE=0
       - SAFE_MODE=0
-    command: yarn start
+    command: yarn start -l=0 -s=0
```

```suggestion
    command: yarn start
```
it should be the same after https://github.com/AcalaNetwork/bodhi.js/pull/585

---

-l=0 -s=0 is the default options